include/bluetooth/hci.h

```diff
@@ -40,8 +40,10 @@ struct bt_hci_evt_hdr {
 #define BT_ACL_CONT                     0x01
 #define BT_ACL_START                    0x02
 
-#define bt_acl_handle(h)                ((h) & 0x0fff)
+#define bt_acl_handle(h)                ((h) & BIT_MASK(12))
 #define bt_acl_flags(h)                 ((h) >> 12)
+#define bt_acl_flags_pb(h)		((h) >> 12 & BIT_MASK(2))
+#define bt_acl_flags_bc(h)		((h) >> 14 & BIT_MASK(2))
```

Comment:
Could you fix the alignment to match the other lines? It seems you're using some tabs, which the other lines don't have

Reply:
The alignment seems to be correct. But sometimes the diff shows it differently, I think it is the + sign in front line that eats up some of the tab-width. Should I replace all the tabs and only use space perhaps? It's what I'd normally do but checkpatch requires 8 spaces to be tab.

Reply:
All the existing bt_acl_* definitions use spaces for the alignment of the values, so please just follow that. The general rule we try to follow (since it makes presentation fairly consistent across different UIs) is tabs for indentation and spaces for alignment, i.e. tabs should only ever exist in the beginning of a line. And yes, as per Linux coding style (which Zephyr inherits) a tab width equals 8 spaces 😃

Reply:
Fixed. Thank you. Just assumed that it was tabs on the other ones.

I know the rule, but I have had issued with checkpatch complaining. But I think that was because I tried to align from the beginning of the line.